pkg/cli/serve/serve.go

```diff
@@ -253,20 +255,32 @@ func (o *serveOptions) Run() error {
 
 	// create a new router
 	r := mux.NewRouter()
+
 	livezHandler := handlers.NewLivezHandler()
 	readyzHandler := handlers.NewReadyzHandler()
 	r.Path(consts.LivezPath).Handler(livezHandler)
 	r.Path(consts.ReadyzPath).Handler(readyzHandler)
-	if o.fleetMode {
-		o.logger.Info("Initialising alertmanager webhook handler in fleet mode")
-		webhookReceiverHandler := handlers.NewWebhookRHOBSReceiverHandler(client, ocmclient)
-		r.Path(consts.WebhookReceiverPath).Handler(webhookReceiverHandler)
-	} else {
-		o.logger.Info("Initialising alertmanager webhook handler in NON-fleet mode")
-		webhookReceiverHandler := handlers.NewWebhookReceiverHandler(client, ocmclient)
-		r.Path(consts.WebhookReceiverPath).Handler(webhookReceiverHandler)
+
+	if err != nil {
```

Comment:
Thanks @hectorakemp
I don't find we are calling the function to fetch internal ID. Do we need this error block?
Overall LGTM

Reply:
@a7vicky - great catch. It's gone now :smile: